client/app/ui-components/tile/custom/item/cui-tile-item-status.component.js

```diff
@@ -55,12 +55,13 @@ angular.module("managerApp")
         scope: true,
         transclude: true,
         template: `
-            <dd class="cui-tile-item-status__description text-truncate d-flex">
-                <cui-status-icon class="cui-tile-item-status__icon" data-type="{{ $ctrl.type }}"></cui-status-icon>
+            <dd class="text-truncate d-flex">
```

Comment:
Maybe use text wrapping instead of `text-truncate` ?

Reply:
This is done :)